### ISSUE TITLE
Cannot issue Attack Ground command on top of friendly units

### ISSUE THREAD

**Issue by sprEEEzy** [beyond-all-reason/Beyond-All-Reason#2226]
### Short description

You can target up to a meter away from friendly units, but not right on top.

Many different use cases where this could be problematic, e.g. trying to issue an attack ground with artillery, but you issue it on a spy bot within the opponents army. No action 

### Problems this will solve

Proposed solution:
Allow targeting ground with no conditions in terms of whos units are close to the target spot.

**Comment by drivver44** [comment]
### information context start
for units that have meaningful aoe for this change to be considered  (AOE that is greater than 32 for structures 24 for units)
for context 
the light laser turrets in both faction have a width of 32 that means aoe would need to be 33 or greater to damage enemy units on the opposing side

ignoring AIR in its entirety 

the smallest unit in the game is 24 width wide via the movedeffs (the equation is simply 16X-8)
`this is specifically about how close a unit can be to anything else on the map nothing more`
the collision box sizes define the movedeffs that a unit can be (ongoing project of mine)
* tangent 
the ongoing project has not really been fleshed out (very little progress made and likely wont need anymore progress made)
the main focus has been on getting edge cases prevented such as this  #2169 
specifically the merl in particular had a case were it would not fire fixable by movedeff size change

### information context end

### a bad suggestion but worth considering
make this behavior specific to units with AOE greater than 32(number is debatable)
what makes this bad is the intricate behaviors being inconsistent in a hard way to remember



### a cleaner solution
if this was taken on a unit type by unit type basis would be the most 
such as arty and bombers which are the primary beneficiaries of these styles of changes
i however believe that only bombers should have this as there are too many varriables to work with here with the other classes
The mess of interceting lines to shapes is not a simple equation that is likely well understood by 2 to 3 of the devs/contributors on bar devs team (me being one of them)( and i mean that in the sense that the outcomes are accurately predictable)

### if all units have this change
this would have severe consequences of friendly fire incidents occurring with no beneficial outcome.
`the beneficial outcome is when an enemy unit gets damaged at all possible expenses of friendly units, which is a very low standard`


### tldr not possible without consiquences of some kind
* this very much is not a simple request as bar features a highly detailed simulation
* the costs of this put into question a balance factor of allowing friendly units to fire through themselves or not ( a key consideration of bars balance)
* player knowledge penalties would be increased as any of the solutions possible have intricacies that would have to be different to the `automated` behaviors

**Comment by sprunk** [comment]
what happens if you hold Alt (either with a right-click, or A then leftclick)? if it doesn't already turn it into attack-ground then perhaps it should. check how it's done in zk
